### PR TITLE
This commit completes ticket 2

### PR DESCRIPTION
**Pull request by MonieW**

### CODE REVIEW

File: db/models/news.models.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const db = require('../connection.js')
+
+exports.selectTopics= ()=> {
+    return db.query(`SELECT * FROM topics`)
+    .then(({rows}) => {
+        console.log(rows)
```
Comment (Keb84) [inline db/models/news.models.js:6]: Please don't forgot to remove any console logs before commiting and pushing 🙂

File: db/controllers/news.controllers.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const{selectTopics} = require('../models/news.models.js')
+
+exports.getTopics = (request, response) => {
+    selectTopics()
+.then((topics) => {
+response.status(200).send({topics})
+
+})
```
Comment (Keb84) [inline db/controllers/news.controllers.js:8]: do you think a catch block would be helpful here 💭

**Review by Keb84 — APPROVED** [review]
Nice work for ticket 2 🤸 please check out my comments and the merge when ready 🙂

File: db/models/news.models.js (hunk 1 above)
Comment (MonieW) [reply db/models/news.models.js:6]: yup, sorry will do

File: db/controllers/news.controllers.js (hunk 2 above)
Comment (MonieW) [reply db/controllers/news.controllers.js:8]: yup, I am confused about catch blocks and errors and whether or not to have a separate file at this stage for errors, but will look at the docs and try to sort for next one, thank you

**Pull request merged**